src/main/webapp/WEB-INF/category.jsp

```diff
@@ -92,7 +92,7 @@
     %>
           <div class="message-div">
             <div class="message-header">
-              User: <%= messages.get(i).getUser() %> -
+              User: <a href="http://localhost:8080/users/<%= messages.get(i).getUser() %>"> <%= messages.get(i).getUser() %> </a> -
```

Comment:
I think the links should link to "/users/"<%= etc, instead of with the localhost. Same with the rest of the links. When we push to appengine, it will probably cause a problem I think.

Reply:
Done